### DISCUSSION TITLE
Running in Windows

### DISCUSSION THREAD

**Discussion by drgerg** [g4ixt/QtTinySA#16] [Show and tell]
![image](https://github.com/g4ixt/QtTinySA/assets/46494738/d3203da4-c830-4562-80eb-98d3d9430eac)

This may be old news, but it seems to work fine in Windows.

**Comment by drgerg** [comment]
I have a compiled executable installer for Windows.  It works here.  With permission, someone can test it.

**Reply by g4ixt** [reply]
Hi Gregory,

Glad it's working for you in Windows.  How big is the Windows executable?  I have built Linux executables but they are near 500Mb and I have been experimenting trying to reduce the size.  Until I have done that I wasn't going to even try building a Windows executable (I rarely use Windows and have not installed Python for it)

Ian

**Comment by g4ixt** [comment]
Windows executable built and in the 'releases' folder
